javascript/apps/taiga/src/app/modules/project/feature-kanban/project-feature-kanban.component.ts

```diff
@@ -261,6 +262,29 @@ export class ProjectFeatureKanbanComponent {
         );
       });
 
+    this.wsService
+      .projectEvents<{ workflowStatus: WorkflowStatus }>(
+        'workflowstatuses.create'
+      )
+      .pipe(untilDestroyed(this))
```

Comment:
Use the new `takeUntilDestroyed` or rx-angular `hold`

Reply:
changed